aws-datastore/src/test/java/com/amplifyframework/datastore/AWSDataStorePluginTest.java

```diff
@@ -206,6 +204,10 @@ public void configureAndInitializeInApiModeWithoutApi() throws JSONException, Am
      */
     @Test
     public void clearStopsSyncUntilNextInteraction() throws AmplifyException, JSONException {
+//        HubAccumulator subscriptionsStarted =
```

Comment:
Is this still needed? Can you remove the commented out stuff?

Reply:
Removed.